File: app/src/main/java/com/BlizzardArmory/model/warcraft/equipment/SpellDescription.java
package com.BlizzardArmory.model.warcraft.equipment;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

/**
 * The type Spell description.
 */
public class SpellDescription {

    @SerializedName("spell")
    @Expose
    private Spell spell;
    @SerializedName("description")
    @Expose
    private String description;
    @SerializedName("display_color")
    @Expose
    private Color color;

    /**
     * Gets spell.
     *
     * @return the spell
     */
    public Spell getSpell() {
        return spell;
    }

    /**
     * Sets spell.
     *
     * @param spell the spell
     */
    public void setSpell(Spell spell) {
        this.spell = spell;
    }

    /**
     * Gets description.
     *
     * @return the description
     */
    public String getDescription() {
        return description;
    }

    /**
     * Sets description.
     *
     * @param description the description
     */
    public void setDescription(String description) {
        this.description = description;
    }

    /**
     * Gets color.
     *
     * @return the color
     */
    public Color getColor() {
        return color;
    }

    /**
     * Sets color.
     *
     * @param color the color
     */
    public void setColor(Color color) {
        this.color = color;
    }

}

File: app/src/main/java/com/BlizzardArmory/model/warcraft/equipment/SpellDescription.kt
package com.BlizzardArmory.model.warcraft.equipment

import androidx.annotation.Keep
import com.BlizzardArmory.model.warcraft.common.Spell
import com.google.gson.annotations.SerializedName


/**
 * The type Spell description.
 */
@Keep
data class SpellDescription(

    @SerializedName("spell")
    var spell: Spell,

    @SerializedName("description")
    var description: String,

    @SerializedName("display_color")
    var color: Color?

)